package.json

```diff
@@ -5,6 +5,7 @@
 		"npm": ">=5.0.0"
 	},
 	"devDependencies": {
+		"@wordpress/browserslist-config": "^2.0.0",
```

Comment:
I'm not happy about that, but this seems to be an issue with the `babel-core` which doesn't follow the following:

> Unfortunately, some tooling does not follow the module resolution spec closely, and instead assumes or requires that dependencies are present specificially in the local node_modules directory. To work around this, it is possible to symlink packages from their hoisted top-level location, to individual package node_modules directory. Lerna does not yet do this automatically, and it is recommended instead to work with tool maintainers to migrate to more compatible patterns.

See more: https://github.com/lerna/lerna/blob/master/doc/hoist.md

This is what I see without this dependency:

> >> Building packages 
a11y
/Users/gziolo/PhpstormProjects/packages/node_modules/babel-core/lib/transformation/file/options/option-manager.js:328
        throw e;
        ^

Error: Cannot find module '@wordpress/browserslist-config' (While processing preset: "/Users/gziolo/PhpstormProjects/packages/node_modules/babel-preset-env/lib/index.js")
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.loadQueries (/Users/gziolo/PhpstormProjects/packages/node_modules/browserslist/node.js:83:19)
    at Function.select (/Users/gziolo/PhpstormProjects/packages/node_modules/browserslist/index.js:632:26)
    at /Users/gziolo/PhpstormProjects/packages/node_modules/browserslist/index.js:151:33
    at Array.reduce (<anonymous>)
    at resolve (/Users/gziolo/PhpstormProjects/packages/node_modules/browserslist/index.js:132:18)
    at browserslist (/Users/gziolo/PhpstormProjects/packages/node_modules/browserslist/index.js:224:16)

Reply:
Hmm, or this is Lerna which doesn't hoist its internal packages? No idea really. @evocateur any ideas why we need to put this dependency also in here?

Reply:
Agreed, less than ideal, not insane, but yeah, having Lerna link these would be 👌

Reply:
Anywhere you need to call a cli in devDependencies, it needs to be specified in the leaf node so lerna knows what to symlink. Hoisting makes it quicker, installing once in the root.

Reply:
Yes, but this one (@wordpress/browserslist-config) is pulled from npm although it is located also inside the packages folder. I hoped to have also the same behavior as in the leaf node for dependencies listed inside the root folder’s package.json. In this case I need to publish package to use the lastest version of it. It makes dogfooding very inconvenient.